### DISCUSSION TITLE
Code scanning reports

### DISCUSSION THREAD

**Discussion by FranciscoGM** [community/community#38791] [Code Security]
### Select Topic Area

Product Feedback

### Body

It would be great to have an option to export reports from Code Scanning findings based on selected filters.
Besides, including the ability to dismiss alerts or flag them as "False positive/Accepted risk/Not applicable".

Thanks!

**Comment by knxp34** [comment]
Suggested format for these would be CSV or JSON.

**Comment by ossanna16** [comment]
Hi there @FranciscoGM and welcome to our community! Thank your the feedback.
